helm/README.md

```diff
@@ -479,27 +497,40 @@ These instructions assume you have
 ### Download the Example Configuration Files
 
 Download the example configuration files from the
-[Voxel51 GitHub](https://github.com/voxel51/fiftyone-teams-app-deploy/helm/gke-examples)
-repository.
+[voxel51/fiftyone-teams-app-deploy](https://github.com/voxel51/fiftyone-teams-app-deploy/helm/gke-examples)
+GitHub repository.
 
 One way to do this might be:
 
 ```shell
-curl -o values.yaml https://raw.githubusercontent.com/voxel51/fiftyone-teams-app-deploy/main/helm/gke-example/values.yaml
-curl -o cluster-issuer.yaml https://raw.githubusercontent.com/voxel51/fiftyone-teams-app-deploy/main/helm/gke-example/cluster-issuer.yaml
-curl -o frontend-config.yaml https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/gke-example/frontend-config.yaml
+curl -o values.yaml \
+  https://raw.githubusercontent.com/voxel51/fiftyone-teams-app-deploy/main/helm/gke-example/values.yaml
+curl -o cluster-issuer.yaml \
+  https://raw.githubusercontent.com/voxel51/fiftyone-teams-app-deploy/main/helm/gke-example/cluster-issuer.yaml
+curl -o frontend-config.yaml \
+  https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/gke-example/frontend-config.yaml
 ```
 
-Update the `values.yaml` file to include
-
-- The Auth0 configuration provided by Voxel51 in `secret.fiftyone`
-- MongoDB username and password in `secret.fiftyone.mongodbConnectionString`
-- `secret.fiftyone.cookieSecret`
-- `secret.fiftyone.encryptionKey`
-- `host` values in `teamsAppSettings.dnsName`
+Update the `values.yaml` file with
+
+- In `secret.fiftyone`
+  - Set the Auth0 configuration provided by Voxel51
+    - `apiClientId`
+    - `apiClientSecret`
+    - `auth0Domain`
+    - `clientId`
+    - `clientSecret`
+    - `organizationId`
```

Comment:
We typically set those when we ship them the values.yaml...

Reply:
Right. Yes. As a newbie, I forgot that stub their values.yaml for them. I'll remove these.